web_frontend/src/types.ts

```diff
@@ -1,7 +1,8 @@
 export type Entity = {
-  name: string;
-  summary: string;
+  name?: string;
```

Comment:
Shouldn't the name be non-nullable?

Reply:
no, insights have no name in the object right now